jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+SYSTEST_CONFIG=$1
+SSH_KEYFILE=$2
+
+bundle exec beaker             \
+  --config "$SYSTEST_CONFIG"   \
+  --helper beaker/helper.rb    \
+  --tests beaker/install/foss/ \
+  --preserve-hosts             \
+  --no-color                   \
+  --xml                        \
+  --ntp                        \
+  --root-keys                  \
+  --repo-proxy                 \
+  --debug                      \
+  --keyfile $SSH_KEYFILE
```

Comment:
We need some additional work here:
1.  We need to pass the name of an existing SUT in to the script.  Something like "perf-bl15"
2.  We need plug that host name into the host.yaml file, or into beaker somewhere.
3.  We need to pass --no-provision

harder stuff.
I think we should CONSIDER using a git submodule to pull in the helper.rb instead of using copy pasta.  BUT, maybe not because git submodule can be scary bad.